tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/listener/ListenerBaloTest.java

```diff
@@ -45,7 +45,7 @@ public void testBasicStructAsObject() {
         final BValue[] result = BRunUtil.invoke(compileResult, "getStartAndAttachCount");
         Assert.assertEquals(result.length, 1, "expected one return type");
         Assert.assertNotNull(result[0]);
-        Assert.assertEquals(result[0].stringValue(), "1_1");
+        Assert.assertEquals(result[0].stringValue(), "2_3");
```

Comment:
Shall we also add a balo test for when the listener type is defined in a separate module?

e.g.,
```ballerina
listener l = new abc:Listener();
```

Reply:
done